deploy/vagrant/bin/spot_request.py

```diff
@@ -82,7 +82,7 @@ def get_host(tag):
 def request_id_to_tag(requests):
     ret = {}
     for i, rid in enumerate([r.id for r in requests]):
-        host = "TachyonMaster" if i == 0 else "TachyonWorker{}".format(i)
+        host = u'TachyonMaster' if i == 0 else "TachyonWorker{}".format(i)
```

Comment:
u stands for unicode.. and that is what aws returns..so for the lookup to work, we have to use a unicode string as the key.